README.md

```diff
@@ -37,75 +37,125 @@ So before diving into AwesomeSplash library, look into the libraries. Especially
 
 # Usage
 
-Add this to your build.grade:
-```javascript
+Add this to your settings.gradle:
+```groovy
 	repositories {
-	//...
-        maven { url "https://jitpack.io" }
+	    //...
+            maven { url "https://jitpack.io" }
     }
 ```
+
 and then in dependencies:
-```javascript
+```groovy
 	dependencies {
-	        compile 'com.github.ViksaaSkool:AwesomeSplash:v1.0.0'
+	        implementation 'com.github.ViksaaSkool:AwesomeSplash:v1.0.0'
 	}
 ```
 
-Here is how you utilze the library in your java code:
-
-```java
+Here is how you utilize the library in your Kotlin code:
+```kotlin
 //extends AwesomeSplash!
-public class YourActivity extends AwesomeSplash {
+class YourActivity : AwesomeSplash() {
 
 	//DO NOT OVERRIDE onCreate()!
 	//if you need to start some services do it in initSplash()!
 
+    override fun initSplash(configSplash: ConfigSplash?) {
+
+	/* you don't have to override every property */
+
+        //Customize Circular Reveal
+        configSplash?.backgroundColor = R.color.purple_500 //any color you want from colors.xml
+        configSplash?.animCircularRevealDuration = 2000 //int ms
+        configSplash?.revealFlagX = Flags.REVEAL_RIGHT  //or Flags.REVEAL_LEFT
+        configSplash?.revealFlagY = Flags.REVEAL_BOTTOM //or Flags.REVEAL_TOP
+
+	//Choose LOGO OR PATH; if you don't provide String value for path it's logo by default
+
+        //Customize Logo
+        configSplash?.logoSplash = R.mipmap.ic_launcher //or any other drawable
+        configSplash?.animLogoSplashDuration = 3000 //int ms
+        configSplash?.animLogoSplashTechnique = Techniques.RotateIn //choose one form Techniques (ref: https://github.com/daimajia/AndroidViewAnimations)
+
+        //Customize Path
+        configSplash?.pathSplash = Constants.DROID_LOGO //set path String
+        configSplash?.originalHeight = 400 //in relation to your svg (path) resource
+        configSplash?.originalWidth = 400 //in relation to your svg (path) resource
+        configSplash?.animPathStrokeDrawingDuration = 3000
+        configSplash?.pathSplashStrokeSize = 3 //I advise value be <5
+        configSplash?.pathSplashStrokeColor = R.color.teal_200 //any color you want from colors.xml
+        configSplash?.animPathFillingDuration = 3000
+        configSplash?.pathSplashFillColor = R.color.white //path object filling color
+
+        //Customize Title
+        configSplash?.titleSplash = "My Awesome App"
+        configSplash?.titleTextColor = R.color.white
+        configSplash?.titleTextSize = 30f //float value
+        configSplash?.animTitleDuration = 3000
+        configSplash?.animTitleTechnique = Techniques.Bounce
+	configSplash?.titleFont = "fonts/myfont.ttf" //provide string to your font located in assets/fonts/
```

Comment:
You can make it better by using extension functions like `configSplash?.let{ }` and have a blocks instead of this java looking bloks :) 

Change that and I'll approve it